apps/site/assets/css/core.scss

```diff
@@ -13,9 +13,11 @@
 @import 'layout-core-shared';
 
 html {
+  width: 100%;
   height: 100%;
   position: relative;
   scroll-behavior: smooth;
+  overflow-x: hidden;
```

Comment:
I'm a bit concerned about this having unintended consequences on other pages, particularly with any other elements that might have or rely on horizontal overflow/scroll behavior. 

So I'm curious what this particular change is meant to do, and whether this is the least-intrusive change we can do to accomplish it?

Reply:
Ah ok, I can omit this bit.  I noticed that some viewport sizes make the homepage have a horizontal scroll when it shouldn't.  I didn't think we had any pages on dotcom where there is horizontal scrolling -- but forgot to account for when the page is zoomed in.  Thanks for catching!